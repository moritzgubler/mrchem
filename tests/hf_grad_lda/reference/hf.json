{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            -4.266952779405394,
            -0.8735732321267697,
            1.2175068299888478
          ],
          "radius": 1.0771438910366888
        },
        {
          "center": [
            -5.9726240875689705,
            -1.4108658430663887,
            3.9542643909246906
          ],
          "radius": 1.4928836384543585
        }
      ],
      "cavity_width": 0.37794522492515403,
      "charge": 0,
      "coords": [
        {
          "atom": "f",
          "xyz": [
            -2.25797417086055,
            -0.462275046496363,
            0.6442768685488515
          ]
        },
        {
          "atom": "h",
          "xyz": [
            -3.1605765564318227,
            -0.7465980517921813,
            2.0925066015625777
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": true,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 5,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -6
    },
    "printer": {
      "file_name": "hf.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 0.001,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 0.001
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 0.001,
          "shared_memory": false,
          "smooth_prec": 0.001
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "svwn5"
              }
            ],
            "spin": false
          }
        }
      },
      "initial_guess": {
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "DFT (SVWN5)",
        "prec": 0.001,
        "restricted": true,
        "type": "core",
        "zeta": 2
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 0.001,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        },
        "geometric_derivative": {
          "geom-1": {
            "operator": "h_nuc_grad",
            "precision": 0.001,
            "smooth_prec": 0.001
          }
        }
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        -2.3034482007428676,
        -0.47659952984111875,
        0.7172401613655945
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 1.719904025547463,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            -0.8972280913620452,
            -0.28411660698606145,
            1.4395587389196445
          ],
          "vector_el": [
            22.585116002812203,
            4.622956863273583,
            -6.451439679582518
          ],
          "vector_nuc": [
            -23.48234409417425,
            -4.907073470259644,
            7.890998418502162
          ]
        }
      },
      "geometric_derivative": {
        "geom-1": {
          "electronic": [
            0.18841208365699463,
            -2.0754989710159437,
            -0.30224686528652456,
            -1.7451139820795545,
            -0.550106829049456,
            2.8002194690324362
          ],
          "electronic_norm": 3.9526975179105786,
          "nuclear": [
            -1.5689195606619417,
            -0.49421531749272585,
            2.5173387449218514,
            1.5689195606619417,
            0.49421531749272585,
            -2.5173387449218514
          ],
          "nuclear_norm": 4.252705427032731,
          "total": [
            -1.380507477004947,
            -2.5697142885086697,
            2.215091879635327,
            -0.17619442141761277,
            -0.05589151155673011,
            0.28288072411058485
          ],
          "total_norm": 3.678322211706672
        }
      },
      "geometry": [
        {
          "symbol": "F",
          "xyz": [
            -2.25797417086055,
            -0.462275046496363,
            0.6442768685488515
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            -3.1605765564318227,
            -0.7465980517921813,
            2.0925066015625777
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -15.989315407026629,
          3.235637551573358,
          4.7527923878469505,
          4.77560198567038,
          4.776109835111442
        ],
        "occupation": [
          2.0,
          2.0,
          2.0,
          2.0,
          2.0
        ],
        "spin": [
          "p",
          "p",
          "p",
          "p",
          "p"
        ],
        "sum_occupied": 3.1016527063510058
      },
      "scf_energy": {
        "E_ee": 94.65104330071759,
        "E_eext": 0.0,
        "E_el": -86.5736125221208,
        "E_en": -329.77091312936926,
        "E_kin": 163.86995613659036,
        "E_next": 0.0,
        "E_nn": 5.2023121410189415,
        "E_nuc": 5.2023121410189415,
        "E_tot": -81.37130038110186,
        "E_x": 0.0,
        "E_xc": -15.323698830059485,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 1,
      "routine": "mrchem.x",
      "total_cores": 1,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 94.65104330071759,
        "E_eext": 0.0,
        "E_el": -86.5736125221208,
        "E_en": -329.77091312936926,
        "E_kin": 163.86995613659036,
        "E_next": 0.0,
        "E_nn": 5.2023121410189415,
        "E_nuc": 5.2023121410189415,
        "E_tot": -81.37130038110186,
        "E_x": 0.0,
        "E_xc": -15.323698830059485,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
