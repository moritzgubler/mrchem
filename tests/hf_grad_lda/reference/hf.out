                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         fastDFT                                  ***
***         Git commit hash    c370a0efe99d190cef5a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Wed Mar 17 16:27:03 2021 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               1
 Total cores             :                                               1
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.0-alpha
 Git branch            : HEAD
 Git commit hash       : 2797a7a18efb69942a6e
 Git commit author     : Stig Rune Jensen
 Git commit date       : Fri Mar 12 12:48:07 2021 +0100

 Linear algebra        : EIGEN v3.3.7
 Parallelization       : OpenMP (1 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 5
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   64.00000    64.00000    64.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -64.00000   -64.00000   -64.00000 ]
 upper bounds          : [   64.00000    64.00000    64.00000 ]
 total length          : [  128.00000   128.00000   128.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       F            :       -2.257974       -0.462275        0.644277
    1       H            :       -3.160577       -0.746598        2.092507
---------------------------------------------------------------------------
 Center of mass          :       -2.303448       -0.476600        0.717240
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize Core Hamiltonian matrix
 Precision               : 1.00000e-03
 Restricted              : True
 AO basis                : Hydrogenic orbitals
 Zeta quality            : 2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               5
 Beta electrons          :                                               5
 Total electrons         :                                              10
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.996812789819e-01
    1    2    p          :                              9.998694302239e-01
    2    2    p          :                              9.999998228187e-01
    3    2    p          :                              9.999999839230e-01
    4    2    p          :                              9.999998001061e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (SVWN5)
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                163.869956136590
 E-N energy              :            (au)               -329.770913129369
 Coulomb energy          :            (au)                 94.651043300718
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                -15.323698830059
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  5.202312141019
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -86.573612522121
 Nuclear energy          :            (au)                  5.202312141019
---------------------------------------------------------------------------
 Total energy            :            (au)             -8.137130038110e+01
                         :      (kcal/mol)             -5.106126149398e+04
                         :        (kJ/mol)             -2.136403182295e+05
                         :            (eV)             -2.214225786689e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -15.989315407027
    1    2    p          :            (au)                  3.235637551573
    2    2    p          :            (au)                  4.752792387847
    3    2    p          :            (au)                  4.775601985670
    4    2    p          :            (au)                  4.776109835111
---------------------------------------------------------------------------
 Sum occupied            :            (au)                  3.101652706351
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       F            :       -2.257974       -0.462275        0.644277
    1       H            :       -3.160577       -0.746598        2.092507
---------------------------------------------------------------------------
 Center of mass          :       -2.303448       -0.476600        0.717240
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                163.869956136590
 E-N energy              :            (au)               -329.770913129369
 Coulomb energy          :            (au)                 94.651043300718
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                -15.323698830059
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  5.202312141019
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -86.573612522121
 Nuclear energy          :            (au)                  5.202312141019
---------------------------------------------------------------------------
 Total energy            :            (au)             -8.137130038110e+01
                         :      (kcal/mol)             -5.106126149398e+04
                         :        (kJ/mol)             -2.136403182295e+05
                         :            (eV)             -2.214225786689e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -15.989315407027
    1    2    p          :            (au)                  3.235637551573
    2    2    p          :            (au)                  4.752792387847
    3    2    p          :            (au)                  4.775601985670
    4    2    p          :            (au)                  4.776109835111
---------------------------------------------------------------------------
 Sum occupied            :            (au)                  3.101652706351
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       22.585116        4.622957       -6.451440
 Magnitude               :            (au)                       23.939095
                         :         (Debye)                       60.847104
---------------------------------------------------------------------------
 Nuclear vector          :      -23.482344       -4.907073        7.890998
 Magnitude               :            (au)                       25.254063
                         :         (Debye)                       64.189420
---------------------------------------------------------------------------
 Total vector            :       -0.897228       -0.284117        1.439559
 Magnitude               :            (au)                        1.719904
                         :         (Debye)                        4.371560
===========================================================================


===========================================================================
                       Geometric Derivative (geom-1)
---------------------------------------------------------------------------
 Total                   :       -1.380507       -2.569714        2.215092
                         :       -0.176194       -0.055892        0.282881
 Norm                    :            (au)                        3.678322
---------------------------------------------------------------------------
 Nuclear                 :       -1.568920       -0.494215        2.517339
                         :        1.568920        0.494215       -2.517339
 Norm                    :            (au)                        4.252705
---------------------------------------------------------------------------
 Electronic              :        0.188412       -2.075499       -0.302247
                         :       -1.745114       -0.550107        2.800219
 Norm                    :            (au)                        3.952698
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m  6s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
